hbase-protocol-shaded/src/main/protobuf/client/Client.proto

```diff
@@ -458,6 +466,13 @@ message RegionAction {
   optional Condition condition = 4;
 }
 
+/*
+* Statistics about the Result's server-side metrics
+*/
+message ResultMetrics {
+  required uint64 blockBytesScanned = 1;
```

Comment:
Not sure what the standard is for HBase, but I know that [optional values with a default are more future proof than required fields](https://stackoverflow.com/a/31814967), and I think that would be better

Reply:
wasn't familiar with the move away from required fields, I'm happy to make this optional.